Utilities/types.h

```diff
@@ -125,6 +125,16 @@ using u16 = std::uint16_t;
 using u32 = std::uint32_t;
 using u64 = std::uint64_t;
 
+using uf8 = std::uint_fast8_t;
+using uf16 = std::uint_fast16_t;
+using uf32 = std::uint_fast32_t;
+using uf64 = std::uint_fast64_t;
+
+using ul8 = std::uint_least8_t;
+using ul16 = std::uint_least16_t;
+using ul32 = std::uint_least32_t;
+using ul64 = std::uint_least64_t;
```

Comment:
these types are useless on x86...

Reply:
I agree, please no. These defines are very weak and more like a headache than something useful. Just as well as usage them in loops. We'd rather remove something unused than adding such things...

Reply:
Not to mention looking like numbers 18, 116, 132, 164, etc...

Reply:
All right, I'll switch to old types later

Reply:
Fix it.
https://github.com/RPCS3/rpcs3/pull/9346/commits/7d1e74d3ff5a19acdad8ba4767eeb9f1df922e4c